views/pages/toplevel/about.njk

```diff
@@ -5,7 +5,6 @@
 
 {% block title %}{{ title }} | {% endblock %}
 
-{% set heroImage = heroImages.mentalHealthFoundation %}
```

Comment:
Worth removing this (and the others) from the hero config file (and the images themselves) to avoid confusion?

Reply:
Yep, but I'll come back and do this afterwards as a bit of cleanup. The images themselves can go then too.